requirements.txt

```diff
@@ -0,0 +1,2 @@
+ansible>=2.4.2
```

Comment:
We have requirement.txt files for the CI tests, but now we need one for the project so that ``notario`` can be easily installed along with our supported ansible version.